control-plane/catalog/to-consul/resource_test.go

```diff
@@ -1712,8 +1712,8 @@ func TestServiceResource_addIngress(t *testing.T) {
 					},
 				},
 				Status: networkingv1.IngressStatus{
-					LoadBalancer: corev1.LoadBalancerStatus{
-						Ingress: []corev1.LoadBalancerIngress{{IP: "1.2.3.4"}},
+					LoadBalancer: networkingv1.IngressLoadBalancerStatus{
```

Comment:
Think this change occurred due to us upgrading the k8s libraries as part of our initial api-gateway work. Merging back main was causing a failure to build until these were updated.